File: backend/Models/Task.cs
namespace backend.Models
{
    public class Task
    {
        public string Name { get; set; }
        public Category Category { get; set; }
        public Score Score { get; set; }
    }

    public enum Category
    {
        Mind,
        Body,
        Soul
    }

    public enum Score
    {
        Low = 1,
        Medium = 2,
        High = 3
    }
}

File: frontend/src/models/Task.ts
export interface Task {
    id: number;
    name: string;
    category: Category;
    score: Score;
}

export enum Category {
    "Mind",
    "Body",
    "Soul"
}

export enum Score {
    Low = 1,
    Medium = 2,
    High = 3
}